code/modules/mob/living/carbon/xenomorph/Xenomorph.dm

```diff
@@ -244,6 +244,7 @@
 	//////////////////////////////////////////////////////////////////
 	var/weedwalking_activated = 0 //Hivelord's weedwalking
 	var/tunnel = 0
+	var/invisible = 0 // for check on lurker invisibility
```

Comment:
no more of these shit vars please

Reply:
changed all of those for boolean and grouped them together